javascript/javascript1/week2/candyHelper.js

```diff
@@ -0,0 +1,36 @@
+//Candy Helper
+const boughtCandyPrices = [];
+const candyPrices = {
+  sweet: 0.5,
+  chocolate: 0.7,
+  toffee: 1.1,
+  "chewing-gum": 0.03,
+};
+function addCandy(candyType, weight) {
+  const pricePerGram = candyPrices[candyType];
+  if (pricePerGram) {
+    const totalPrice = pricePerGram * weight;
+    boughtCandyPrices.push(totalPrice);
+  } else {
+    console.log("Invalid candy type");
+  }
+}
+const amountToSpend = Math.random() * 100;
```

Comment:
You can move this variable inside of the function, to limit it's scope from global to local

Reply:
Moved variable inside the function...